plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -285,6 +286,27 @@ const eachMessage =
                             event.ip,
                             event.properties || {}
                         )
+                        // the replay record timestamp has to be valid and be within a reasonable diff from now
+                        if (replayRecord !== null) {
+                            const asDate = DateTime.fromSQL(replayRecord.first_timestamp)
+                            if (!asDate.isValid || Math.abs(asDate.diffNow('months').months) >= 0.99) {
+                                captureMessage(
```

Comment:
How often does this happen? Are we in danger of sending a ridiclous number of events to Sentry? 
I'm wondering if this makes more sense as a prometheus metric?

Reply:
We'd have sent about a 1000 to date. Problem with a prom metric is I want to see the message not just that it happened.